fusequery/query/src/interpreters/plan_scheduler_test.rs

```diff
@@ -186,88 +117,79 @@ async fn test_scheduler_plan_with_convergent_and_expansive_stage() -> Result<()>
      *        +-------->|RemotePlan +------>|SelectPlan +-----------+
      *                  +-----------+       +-----------+
      */
-    let (context, _cluster) = create_env().await?;
-    let scheduled_actions = PlanScheduler::reschedule(
-        context.clone(),
-        HashMap::<String, bool>::new(),
-        &PlanNode::Select(SelectPlan {
-            input: Arc::new(PlanNode::Stage(StagePlan {
-                kind: StageKind::Convergent,
-                scatters_expr: Expression::Literal(DataValue::UInt64(Some(0))),
-                input: Arc::new(PlanNode::Select(SelectPlan {
-                    input: Arc::new(PlanNode::Stage(StagePlan {
-                        kind: StageKind::Expansive,
-                        scatters_expr: Expression::ScalarFunction {
-                            op: String::from("blockNumber"),
-                            args: vec![],
-                        },
-                        input: Arc::new(PlanNode::Empty(EmptyPlan::create())),
-                    })),
+    let context = create_env().await?;
+    let scheduler = PlanScheduler::try_create(context.clone())?;
+    let scheduled_tasks = scheduler.reschedule(&PlanNode::Select(SelectPlan {
+        input: Arc::new(PlanNode::Stage(StagePlan {
+            kind: StageKind::Convergent,
+            scatters_expr: Expression::Literal(DataValue::UInt64(Some(0))),
```

Comment:
What does scatters_expr mean here, why is it 0?

Reply:
We use it to shard data, for example:
```
Input  data column: [1, 1, 2, 3, 2, 5]
Input scatter column(eval by scatters expr) : [0, 1, 0, 0, 1, 1]
It will return &[Column; 2]:
    - Output column 1: [1, 2, 3]
    - Output column 2: [1, 2, 5]
```

Reply:
In cluster mode, the output columns will be sent to different nodes

Reply:
The reason why scatters_epxr is 0 here is that the stage is of Convergent kind?

Reply:
Yep.